rllib/agents/cql/cql.py

```diff
@@ -31,7 +27,7 @@
         # Offline RL does not need IS estimators.
         "input_evaluation": [],
         # Number of iterations with Behavior Cloning Pretraining.
-        "bc_iters": 20000,
+        "bc_iters": 2000,
```

Comment:
@michaelzhiluo , can you check whether this makes more sense now?
Now that we count iterations differently (one iteration == sampling from buffer + update), I thought lowering this would make more sense.